tests/test-cluster.py

```diff
@@ -463,3 +464,119 @@ def test_no_cert_reissue_in_nodes(self):
         for vm in self.VM:
             lock_files = vm.run("ls /var/snap/microk8s/current/var/lock/")
             assert "no-cert-reissue" in lock_files.decode()
+
+
+class TestUpgradeCluster(object):
+    @pytest.fixture(autouse=True, scope="module")
+    def setup_old_versioned_cluster(self):
+        """
+        Provision VMs of the previous version and form the cluster.
+        """
+        try:
+            print("Setting up cluster of an older version")
+            track, risk_level = channel_to_test.split('/')
+            if track == "latest":
+                track = "1.26"   # TODO: remove this hardcoding
+            track = float(track) - 0.01
+            older_version = str(track) + '/' + risk_level
+            print("Old version is {}".format(older_version))
+
+            type(self).VM = []
+            if not reuse_vms:
+                size = 3
```

Comment:
We do not need an HA cluster for this test. A single node would be enough.

Reply:
Changing size to 1